src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts

```diff
@@ -91,4 +85,13 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
       this.dtTrigger.next();
     }
   }
+
+  openURLInNewTab(uri: string): WindowProxy | string {
+    return this.isURL(uri) ? window.open(uri, '_blank') : '';
+  }
+
+  isURL(uri: string) {
+    const regex = new RegExp('http*', 'g');
+    return regex.test(uri) ? true : false;
```

Comment:
you could return `regex.test(uri)` directly, as it already returns true or false.

Reply:
Also, a type in the function may help. `isURL(uri: string) : boolean`

Reply:
Yes, the type of the function has not yet been placed.

Reply:
Nice ! 🌟